### DISCUSSION TITLE
Not sure if this option is obsolete. Please clarify.

### DISCUSSION THREAD

**Discussion by knightswatch3** [webpack/webpack#13047] [Q&A]
My whole project's migrated to v5 of webpack. But when I am trying to import any libraries in my react code, I get the following:

Field 'browser' doesn't contain a valid alias configuration
        /root/Myproj/src/node_modules/react-bootstrap doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /root/Myproj/src/node_modules/react-bootstrap.js doesn't exist
      .json
        Field 'browser' doesn't contain a valid alias configuration
        /root/Myproj/src/node_modules/react-bootstrap.json doesn't exist
      .wasm
        Field 'browser' doesn't contain a valid alias configuration
        /root/Myproj/src/node_modules/react-bootstrap.wasm doesn't exist
      as directory
        /root/Myproj/src/node_modules/react-bootstrap doesn't exist

Most of the online resources suggest the fix in the path in resolve and entry-points. But everything looks fine. The code throws the error only when importing them in the my react components.

Eg:

> **import {Nav} from "react-bootstrap";**

**Comment by alexander-akait** [comment]
Please install `react-bootstrap`

**Reply by knightswatch3** [reply]
Ofcourse I did install react-bootstrap and bootstrap. But still I get this error.

**Reply by alexander-akait** [reply]
Please provide configuration

**Reply by knightswatch3** [reply]
![image](https://user-images.githubusercontent.com/9092073/113493137-adae9280-9491-11eb-8dbc-192af96f05ff.png)


That is how my configuration is setup.

**Reply by sokra** [reply]
You have overriden the default `resolve.modules` configuration, so you basically disabled resolving from `node_modules`.
Make sure to reference the default with `"..."` by adding that to the array.
